plugins/dynamodb/plugin.spec.yaml

```diff
@@ -104,6 +104,58 @@ connection:
     example: us-east-1
     default: us-east-1
 actions:
+  get_item:
+    title: GetItem
+    description: The GetItem operation returns a set of attributes for the item with the given primary key. If there is no matching item, GetItem does not return any data and there will be no Item element in the response 
+    input:
+      table_name:
+        title: Table Name
+        description: The table name to search
+        type: string
+        required: true
+        example: Table-name
+      key:
+        title: Key 
+        description: A map of attribute names to AttributeValue objects, representing the primary key of the item to retrieve
+        type: object
+        required: true
+        example: '{"CustomerID": {"S":"12345"}}'
+      consistent_read:
+        title: Consistent Read
+        description: "Determines the read consistency model: If set to true , then the operation uses strongly consistent reads; otherwise, the operation uses eventually consistent reads"
```

Comment:
```suggestion
        description: Determines the read consistency model; If set to true, then the operation uses strongly consistent reads; otherwise, the operation uses eventually consistent reads
```